tests/testthat/test-tm_g_ci.R

```diff
@@ -1,3 +1,250 @@
+app_driver_tm_g_ci <- function() {
+  ADSL <- tmc_ex_adsl
+  ADLB <- tmc_ex_adlb
+
+  init_teal_app_driver(
+    data = cdisc_data(
+      ADSL = ADSL,
+      ADLB = ADLB,
+      code = "
+        ADSL <- tmc_ex_adsl
+        ADLB <- tmc_ex_adlb
+      "
+    ),
+    modules = modules(
+      tm_g_ci(
```

Comment:
there is only one modules, so maybe no need to wrap it under `modules` function?
```suggestion
    modules = tm_g_ci(
```